runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsPlugin.java

```diff
@@ -133,4 +137,22 @@ else if (client.getTickCount() - lastCombatActionTickCount >= config.hideAfterCo
 			barsDisplayed = false;
 		}
 	}
+
+	@Subscribe
+	public void onWidgetClosed(WidgetClosed ev)
```

Comment:
This works fine while the plugin is running, but does not work if the plugin is toggled on while in an LMS match.

Reply:
Fixed now, when the plugin turns on it checks if the widget is open

Reply:
On second thought, `client.getWidget()` is actually very fast. This could be moved to the overlay instead, making the change into:

```java
private boolean inLms()
{
	return client.getWidget(WidgetInfo.LMS_KDA) != null;
}
```

as a method in the overlay class, with no plugin class changes needed.